deploy/operator.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: apps/v1
```

Comment:
Why do we need this file in our repo?

Reply:
As far as I remember the files are needed solely for the `operator-sdk olm-catalog gen-csv --csv-version $new_version --from-version $old_version` command to generate newer csv. If they are not present, the command won't run. I remember I was experimenting with removing some of the files - not all are obligatory (as far as I remember) but I can't recall which ones.

Reply:
But do we need to commit them to our repo? Can't we just generate the files and push them to the https://github.com/operator-framework/community-operators repo?